std/container/slist.d

```diff
@@ -285,6 +285,27 @@ Complexity: $(BIGOH 1)
     }
 
 /**
+Reverses SList in-place. Performs no memory allocation.
+
+Complexity: $(BIGOH n)
+     */
+    void reverse()
+    {
+        if (!empty)
+        {
+            Node* prev;
```

Comment:
Might want to explicitly initialize `prev` to null here, otherwise it looks like the loop body is using an uninitialized variable.